src/DynamoCoreWpf/ViewModels/PackageManager/PackageViewModel.cs

```diff
@@ -233,24 +256,125 @@ private void Uninstall()
             }
             finally
             {
-                RaisePropertyChanged(nameof(PackageLoadStateTooltip));
-                RaisePropertyChanged(nameof(PackageLoadStateText));
+                NotifyLoadStatePropertyChanged();
             }
         }
 
         private bool CanUninstall()
         {
             if (!Model.InUse(dynamoViewModel.Model) || Model.LoadedAssemblies.Any())
             {
-                return Model.BuiltInPackage ? 
+                return Model.BuiltInPackage ?
                     Model.LoadState.State != PackageLoadState.StateTypes.Unloaded &&
                     Model.LoadState.ScheduledState != PackageLoadState.ScheduledTypes.ScheduledForUnload :
                     Model.LoadState.ScheduledState != PackageLoadState.ScheduledTypes.ScheduledForDeletion;
             }
             return false;
         }
 
-        private void GoToRootDirectory()
+        // Loads a built-in package that was previously set as Unloaded
+        private void Load()
+        {
+            var dynModel = dynamoViewModel.Model;
+            var packageLoader = dynModel.GetPackageManagerExtension().PackageLoader;
+            var conflicts = packageLoader.LocalPackages.ToList().Where(x => x.Name == Model.Name && x != Model);
```

Comment:
does `package` override equals correctly or do you wish to use reference equality, if so, maybe use ReferenceEquals explicitly?

Reply:
From my understanding, by default, the referenceEquals is used by equality operators on objects 
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/equality-operators#reference-types-equality
The Package class does not override or overload anything related to equality comparers.
But I agree that it is a bit unclear...and always triggers an alarm. We can always override the Equality methods in the Package class to make things clearer...